data/json/items/gun/223.json

```diff
@@ -151,138 +149,82 @@
     "weight": 2500,
     "volume": 6,
     "price": 350000, "//" : "Current milspec, so Expensive.",
-    "bashing": 12,
-    "to_hit": -1,
     "material": ["steel", "plastic"],
     "symbol": "(",
     "color": "dark_gray",
     "skill": "rifle",
     "ammo": "223",
-    "ranged_damage": -2,
-    "range": -12,
-    "dispersion": 150,
     "sight_dispersion": 60,
     "aim_speed" : 6,
-    "recoil": 60,
+    "recoil": -150,
     "durability": 3,
```

Comment:
It also has very low `durability` in-keeping with the description. Once we rework jams/misfires that's a harsh penalty whilst still making the gun usable which is perhaps more interesting than just making it a bad item.